generation/update_googleapis_commit.sh

```diff
@@ -74,10 +74,14 @@ popd
 rm -rf tmp-googleapis
 sed -i -e "s/^googleapis_commitish.*$/googleapis_commitish: ${latest_commit}/" "${generation_config}"
 
-
 git add "${generation_config}"
-# use --allow-empty because (rarely) there's no change.
-git commit --allow-empty -m "${title}"
+changed_files=$(git diff --cached --name-only)
```

Comment:
This logic is similar to 
https://github.com/googleapis/google-cloud-java/blob/5b27334e192742b2c3be1276618c65ce456c1717/generation/hermetic_library_generation.sh#L101